guides/common/modules/ref_custom-provisioning-snippet-example-for-debian.adoc

```diff
@@ -0,0 +1,17 @@
+[id="Custom_Provisioning_Snippet_Example_for_Debian_{context}"]
+= Custom Provisioning Snippet Example for Debian
+
+You can use `Custom Post` snippets to call external APIs from within the provisioning template directly after provisioning a hosts.
+
+.Custom Post Hook Example for Debian/Ubuntu
```

Comment:
Should we put the name of the custom snippet to make it clear where the snippet is added?

Reply:
Added.